_layouts/page.html

```diff
@@ -58,6 +58,9 @@
 		{% include header.html %}
 
 		<main role="main">
+
+			{% include flash.html %}
```

Comment:
oh, good question. i hadn't really thought about that. I put it in the main page include because generally flash messages should be available to every page, but that doesn't necessarily imply that every page *should* have one. So, erm. I don't know. 🤔